hatchet_sdk/client.py

```diff
@@ -58,32 +94,5 @@ def with_host_port_impl(config: ClientConfig):
     return with_host_port_impl
 
 
-def new_client(defaults: ClientConfig = ClientConfig(), *opts_functions) -> ClientImpl:
-    config: ClientConfig = ConfigLoader(".").load_client_config(defaults)
-
-    for opt_function in opts_functions:
-        opt_function(config)
-
-    if config.tls_config is None:
-        raise ValueError("TLS config is required")
-
-    if config.host_port is None:
-        raise ValueError("Host and port are required")
-
-    conn: grpc.Channel = new_conn(config)
-
-    # Instantiate client implementations
-    event_client = new_event(conn, config)
-    admin_client = new_admin(config)
-    dispatcher_client = new_dispatcher(config)
-    rest_client = RestApi(config.server_url, config.token, config.tenant_id)
-    workflow_listener_client = None
-
-    return ClientImpl(
-        event_client,
-        admin_client,
-        dispatcher_client,
-        workflow_listener_client,
-        rest_client,
-        config,
-    )
+new_client = ClientImpl.new_from_environment
+new_client_raw = ClientImpl.new_from_config
```

Comment:
```suggestion
new_client_from_config = ClientImpl.new_from_config
```

improve readability?

Reply:
sorry not sure i understand the comment here